nf_core/modules/test_yml_builder.py

```diff
@@ -170,7 +170,7 @@ def build_single_test(self, entry_point):
             mod_name_parts = self.module_name.split("/")
             tag_defaults = []
             for idx in range(0, len(mod_name_parts)):
-                tag_defaults.append("_".join(mod_name_parts[: idx + 1]))
+                tag_defaults.append("/".join(mod_name_parts[: idx + 1]))
```

Comment:
So this will change the `tag` in `test.yml`?

Reply:
To `bwa/index` instead of `bwa_index`?

Reply:
yep, that should do the trick.